### ISSUE TITLE
About reproducing published experiment results

### ISSUE THREAD

**Issue by tnt-ooo-tnt** [DeadAt0m/ActiveSparseShifts-PyTorch#1]
It is really honor to study on the common research topic of you.

I've got a few questions about reproducing your works.

1) You have mentioned "the initial learning rate starts from 0.6 and linear decaying policy had been applied and it stops after 480 epochs while the training of SSL is stopped after 240 epochs."
I know it is a shame of me but it is quite complex to understand what you have exactly done.
I hope you to clarify this one for me. Frankly, I want to know the value of all hyper-parameters that you used.

2) You also have mentioned that "To further rich the training images, more image of distortions are provided as used in Inception training[33, 10]. But it will be withdrawn in last several epochs."

I had a look at the referenced papers, but I could't find the information about distorted images.
So, please let me know what distortions mean, and when is the precise timing to withdraw them.

* I am not a good English speaker, so there might be some rude expression. Please excuse me. Thank you.

**Comment by DeadAt0m** [comment]
Hello, thank you for interest to this repo.

I am not author of  original article, I just implement  this for my own purposes. So I am not intended to reproduce the article itself. 

However, in future I plan to add the comparisons of Shifts using on toy examples e.g. MobileNetv2 on CIFAR
